cmd/query/app/server.go

```diff
@@ -274,10 +274,13 @@ func (s *Server) initListener() (cmux.CMux, error) {
 func (s *Server) Start() error {
 	cmuxServer, err := s.initListener()
 	if err != nil {
-		return err
+		return fmt.Errorf("query server failed to initialize listener: %w", err)
 	}
 	s.cmuxServer = cmuxServer
 
+	// Logging for ports
+	s.logger.Info("Initialized listener", zap.String("HTTP", s.queryOptions.HTTPHostPort), zap.String("gRPC", s.queryOptions.GRPCHostPort))
```

Comment:
is there a value in this log? From the ticket description, the ports are already logged
```
    logger.go:146: 2024-06-03T15:06:55.395Z	INFO	Starting GRPC server	{"port": 16686, "addr": ":16686"}
    logger.go:146: 2024-06-03T15:06:55.395Z	INFO	Starting HTTP server	{"port": 16686, "addr": ":16686"}
    logger.go:146: 2024-06-03T15:06:55.395Z	INFO	Starting CMUX server	{"port": 16686, "addr": ":16686"}
    logger.go:146: 2024-06-03T15:07:05.399Z	INFO	GRPC server stopped	{"port": 16686, "addr": ":16686"}
```

Reply:
I will remove this redundant log statement to avoid unnecessary duplication.